pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -107,10 +108,14 @@
 }
 
 
+def _default_2q_gate(device_name: str) -> OpType:
+    # If we change this, we should update the main documentation page and highlight it
+    # in the changelog.
+    return OpType.ZZPhase
```

Comment:
I may be leaning towards a static default again, i.e. this, because:
• It sounds like a dynamic default may change as regularly as day to day
• The optimal gate is likely circuit dependent, meaning the dynamic default is likely inaccurate
• So, if dynamic, we're providing inconsistent, and possibly confusing, compilation for the sake of a metric that is likely not accurate
• This is assuming that `ZZPhase` is a reasonable default choice for a long (~many months) time
• This is also assuming fidelity information for operations becomes available (which I believe it is?)
• The documentation is clear regarding how to recompile, so savvy users should easily be able to do so


We should also consider looking at a new noise aware compilation method for these devices that considers circuit structure and gate fidelities when picking a target 2qb gate.